### PR TITLE
regmap size check updated

### PR DESCRIPTION
**Pull request by sarab96**
regmap size check updated.
Change is based on another PR for cpp XRT kernel APIs
No change in functionality

### CODE REVIEW

File: src/xma/src/xmaapi/xmaxclbin.cpp [hunk 1]
```diff
@@ -119,12 +119,17 @@ static int get_xclbin_iplayout(const char *buffer, XmaXclbinInfo *xclbin_info)
...
+            std::copy_if(args.begin(), args.end(), std::back_inserter(args2), 
+                  [](const xarg& y){return y.index != no_index;});
             temp_ip_layout.arg_start = -1;
             temp_ip_layout.regmap_size = -1;
-            if (args.size() > 0) {
-                temp_ip_layout.arg_start = args[0].offset;
-                auto& last = args.back();
+            if (args2.size() > 0) {
+                temp_ip_layout.arg_start = args2[0].offset;
+                auto& last = args2.back();
```
Comment (stsoe) [inline src/xma/src/xmaapi/xmaxclbin.cpp:132]: Appears to assume that offset is  increasing with argument.  Not sure if that is a safe assumption, but it is definitely an implicit assumption that should be commented.

Comment (sarab96) [reply src/xma/src/xmaapi/xmaxclbin.cpp:132]: I see sort in get_kernel_arguments function. Line# 606. So this should be fine.

Comment (sarab96) [reply src/xma/src/xmaapi/xmaxclbin.cpp:132]: On offset increasing by arg index. Yes that is an assumption. But that is fair assumption. It probably can be confirmed by v++ or hls team. Also I have not seen any place where this isn't true.

Comment (sarab96) [reply src/xma/src/xmaapi/xmaxclbin.cpp:134]: Comment on this included in the code

**Pull request merged**